Source/SocketAckEmitter.swift

```diff
@@ -44,6 +44,10 @@ public final class SocketAckEmitter : NSObject {
         
         socket.emitAck(ackNum, with: items)
     }
+        
+    public func isRequired() -> Bool {
+        return ackNum != -1;
```

Comment:
in the JavaScript API, when an event expecting an ack is sent by server, the client provides the ack callback as last argument in event handler.
not a way to "require" an ack .. more like "an ack is expected"